### DISCUSSION TITLE
Particle Temperature

### DISCUSSION THREAD

**Discussion by r-broek** [firemodels/fds#12943] [Q&A] [answered]
Does the PARTICLE TEMPERATURE (User Guide 22.9.4) for a massive particle refer to the surface temperature of the particle? Or perhaps some average over its volume is used?

**Comment by rmcdermo** [comment, marked as answer]
It is the surface temperature, which for thermally thin droplets represents the temperature of the whole particle.  For thermally thick particles, it is just the surface temperature.

**Reply by r-broek** [reply]
Thanks!
